app/scripts/controllers/network/networks.js

```diff
@@ -18,6 +20,18 @@ networks.networkList = {
     'exchanges': ['ShapeShift'],
     'buyUrl': '',
   },
+  [MORDEN]: {
+    'chainId': MORDEN_CODE,
+    'ticker': 'ETC',
+    'blockExplorerTx': 'http://mordenexplorer.ethertrack.io/tx/[[txHash]]',
+    'blockExplorerAddr': 'http://mordenexplorer.ethertrack.io/addr/[[address]]',
+    'blockExplorerToken': 'http://mordenexplorer.ethertrack.io/token/[[tokenAddress]]/[[address]]',
+    'service': 'Gastracker.io',
+    'rpcUrl': 'https://etc-geth.0xinfra.com',
```

Comment:
`FIXME`  is there any official api-server/block explorer exist for the Morden testnet ?

Reply:
Maybe contact the ETC rpc providers about adding support for the testnet. If there isn't a reliable rpc provider for the ETC testnet, don't list it among the supported networks.

Reply:
@hackmod Remove modern testnet, no one is using ethereum classic testnet for testing purpose. Why do we need the option??

Reply:
OK. as you already know, this is a quick hack and I'm not much familiar with a react program.
we need more testing to find possible bugs. that's the reason the morden testnet added.

Reply:
temporary added morden api server.
but the api server provided by other coin project  seems improper to me.

Reply:
@hackmod Then remove useless morden testnet menu, we've added the endpoint only for this PR merge if you don't need it we will close then

Reply:
removed and rebased